localization/replace/english/decisions_l_english.yml

```diff
@@ -0,0 +1,4 @@
+﻿l_english:
+ pet_dog_decision_desc:0 "It is time to take a deep breath and find my trusted [ROOT.Char.Custom('CompanionLegAmount')] companion [ROOT.Char.Custom('DogStoryName')]. Some people say [ROOT.Char.Custom('CompanionSpecies')|l]s are base creatures, but no matter how tough life gets I know [ROOT.Char.Custom('DogStorySheHe')] will always be there for me!"
+ pet_dog_decision_tooltip:0 "Pet your [ROOT.Char.Custom('CompanionSpecies')|l]"
```

Comment:
Why it's in `l` style?

Reply:
Because the species needs to be Camel Case in the titles and lower case in the descriptions